{
	"schema_version": "1.6.0",
	"id": "RHSA-2020:5435",
	"modified": "2024-11-22T15:07:21Z",
	"published": "2024-09-16T04:14:47Z",
	"related": [
		"CVE-2020-14019"
	],
	"summary": "Red Hat Security Advisory: python-rtslib security update",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:L/A:L"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:7::client",
				"name": "python-rtslib",
				"purl": "pkg:rpm/redhat/python-rtslib"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:2.1.74-1.el7_9"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:5435.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:7::client",
				"name": "python-rtslib-doc",
				"purl": "pkg:rpm/redhat/python-rtslib-doc"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:2.1.74-1.el7_9"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:5435.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:7::computenode",
				"name": "python-rtslib",
				"purl": "pkg:rpm/redhat/python-rtslib"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:2.1.74-1.el7_9"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:5435.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:7::computenode",
				"name": "python-rtslib-doc",
				"purl": "pkg:rpm/redhat/python-rtslib-doc"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:2.1.74-1.el7_9"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:5435.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:7::server",
				"name": "python-rtslib",
				"purl": "pkg:rpm/redhat/python-rtslib"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:2.1.74-1.el7_9"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:5435.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:7::server",
				"name": "python-rtslib-doc",
				"purl": "pkg:rpm/redhat/python-rtslib-doc"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:2.1.74-1.el7_9"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:5435.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:7::workstation",
				"name": "python-rtslib",
				"purl": "pkg:rpm/redhat/python-rtslib"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:2.1.74-1.el7_9"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:5435.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:7::workstation",
				"name": "python-rtslib-doc",
				"purl": "pkg:rpm/redhat/python-rtslib-doc"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:2.1.74-1.el7_9"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:5435.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2020:5435"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#moderate"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1854723"
		},
		{
			"type": "ADVISORY",
			"url": "https://security.access.redhat.com/data/csaf/v2/advisories/2020/rhsa-2020_5435.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2020-14019"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2020-14019"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-14019"
		}
	]
}
